### PR TITLE
Kathleen

### PR DESCRIPTION
**Pull request by kdeardorff2**
Restructured agile section of the bootcamp.

### CODE REVIEW

**Comment by kdeardorff2** [comment]
Will fix that right now, great idea!

**Comment by kdeardorff2** [comment]
This should do the trick!

File: docs/_sidebar.md [hunk 1]
```diff
@@ -24,16 +24,35 @@
...
-- [3.3 - Version Control](3-software-development/version-control.md)
-  - [3.3.1 - Git](3-software-development/version-control_git.md)
-  - [3.3.2 - GitHub](3-software-development/version-control_github.md)
-  - [3.3.3 - Deliverables](3-software-development/version-control_deliverables.md)
-- [3.4 - Pair Programming](3-software-development/pair-programming.md)
-- [3.5 - Testing](3-software-development/testing.md)
-- [3.5 - Hello DevOps](3-software-development/hello-devops.md)
+- [3.0 - Development Practices](3/overview.md)
+- [3.1 - Agile](3/agile.md)
+<<<<<<< HEAD
```
Comment (ChrisSchreiber) [inline docs/_sidebar.md:29]: Looks like there was a merge conflict that still needs to be resolved.

Comment (kdeardorff2) [reply docs/_sidebar.md]: Fixed the merge conflict in the latest commit.

**Pull request merged**